webui/src/components/editrecord.jsx

```diff
@@ -99,6 +99,9 @@ const EditRecord = React.createClass({
             errors: {},
             dirty: false,
             waitingForServer: false,
+            readOnly: false,
+            checkboxValue: null,
```

Comment:
not a good name, which checkbox? is it about the confirmation for editing?

Reply:
The checkbox at the end of the edit draft. When it's true then the draft will be either sent for review or it's ready for publication. Maybe `review_or_publish`?

Reply:
yes, I would call it `reviewOrPublishConfirmed`